README.md

```diff
@@ -13,20 +13,21 @@ Currently requires
 
 ## Quick Start Instructions
 
-As root: `start.py [-h] [-c] [--max-args MAX_ARGS] [-j] [-e EVENTS_TO_TRACE]`
+As root: `tracee.py [-h] [-c] [--max-args MAX_ARGS] [-j] [-e EVENTS_TO_TRACE]`
 
 optional arguments:
-  -h, --help            show this help message and exit
-  -c, --container       only trace newly created containers
-  --max-args MAX_ARGS   maximum number of arguments parsed and displayed, defaults to 20
-  -j, --json            save events in json format
-  -e EVENTS_TO_TRACE, --events-to-trace EVENTS_TO_TRACE
-                        trace only the specified events and syscalls (default: trace all)
-
-examples:
-    ./start.py -c
-
-Following is an output example of Tracee after running
+&nbsp;&nbsp;-h, --help            show this help message and exit<br>
+&nbsp;&nbsp;-c, --container       only trace newly created containers<br>
+&nbsp;&nbsp;--max-args MAX_ARGS   maximum number of arguments parsed and displayed, defaults to 20<br>
+&nbsp;&nbsp;-j, --json            save events in json format<br>
+&nbsp;&nbsp;-l, --list            list events<br>
+&nbsp;&nbsp;-e EVENTS_TO_TRACE, --events-to-trace EVENTS_TO_TRACE<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;trace only the specified events and syscalls (default: trace all)
```

Comment:
Do we need the nbsp entity? I don't recall seeing them in readme's before, and it would be easier to manage to just use the regular space

Reply:
Is there an easier way to indent md? Or do you suggest to remove the indentation? I personally think it's easier to read with indentation

Reply:
The normal way if you want to force indentation would be to do it as a code block 
```
  like this
```
and then indentation is respected